clients/v1_shell/v1_shell.go

```diff
@@ -621,6 +624,22 @@ func (c *Client) outputColumns(response api.InfluxqlJsonResponse) {
 	writer.Flush()
 }
 
+func (c *Client) outputTable(response api.InfluxqlJsonResponse) {
+	for _, res := range response.GetResults() {
```

Comment:
Should we create a new window for every table? For queries that create a bunch of small tables this seems awkward.

Reply:
When the table is small enough to fit on screen with no scrolling, there is no interactive view and it prints all the tables in one go. Here's an example:
<img width="422" alt="Screen Shot 2022-06-23 at 11 25 14 AM" src="https://user-images.githubusercontent.com/32912555/175358693-08e0e8a0-3981-492b-9109-f45f2da2ab44.png">
What do you think of that behavior?